src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,58 @@
 package core.basesyntax;
 
-public class WorkWithFile {
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
 
+public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        ArrayList<DataFromFile> dataList = new ArrayList<>();
+        getDataFile(fromFileName, dataList);
+        writeToFile(toFileName, calculating(dataList));
+    }
+
+    private void getDataFile(String fromFileName, ArrayList<DataFromFile> dataList) {
+        try (BufferedReader reader = new BufferedReader(new FileReader(fromFileName))) {
+            String line = null;
+            DataFromFile data = null;
+            while ((line = reader.readLine()) != null) {
+                String[] strings = line.split(",");
+                data = new DataFromFile(strings[0], Integer.valueOf(strings[1]));
```

Comment:
```suggestion
                data = new DataFromFile(strings[OPERATION_INDEX], Integer.valueOf(strings[AMOUNT_INDEX]));
```
create constants for these indices